cafebabel/templates/base.html

```diff
@@ -80,6 +64,36 @@
 <a href={{ lang_url_for('proposals.create') }} id=participate>Participate</a>
 <a href={{ lang_url_for('articles.to_translate') }} id=translate>Translate</a>
 
+<button class="menu profile">
+  {% if current_user.is_authenticated %}
+    {#{{ profile_image(user) }}#}
```

Comment:
Here I would like to use this macro. But neither `user` nor `author` is defined. How should I enable `user` so that I can get profile pic on every pages @davidbgk ?

Reply:
Did you try with `current_user`?